code-challenges.test.js

```diff
@@ -25,7 +28,26 @@ const colors2 = [
   "aquamarine",
   "saffron"
 ]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+// Psedocode 1. Create a function called firstItem with array as a parameter
+//           2. Input is a function that takes in an array 
+//           3. Use .shift to remove the first element from an array 
+//           4. Use .sort to return array in order with math.random to shuffle the order.
+//           5. Output returns the removes the first item from the array and shuffles the remaining content.
+const firstItem = (array) => {
+   array.shift("")
+   return array.sort((a, b) => Math.random())}
+console.log(firstItem(colors1))
+console.log(firstItem(colors2))
+
+describe("firstItem", () => {
+  it("removes the first item from the array and shuffles the remaining content", () => {
+    expect(firstItem(colors1)).toEqual( expect.arrayContaining(firstItem(colors1)))
+    expect(firstItem(colors2)).toEqual( expect.arrayContaining(firstItem(colors2)))
+  
```

Comment:
Your expect should contain the desired output . You wouldn't want to use the variable that includes a value you don't want.